src/DBFTPlugin/ConsensusService.cs

```diff
@@ -411,8 +411,9 @@ private void OnPrepareRequestReceived(ExtensiblePayload payload, PrepareRequest
             if (context.RequestSentOrReceived || context.NotAcceptingPayloadsDueToViewChanging) return;
             if (message.ValidatorIndex != context.Block.PrimaryIndex || message.ViewNumber != context.ViewNumber) return;
             if (message.Version != context.Block.Version || message.PrevHash != context.Block.PrevHash) return;
+            if (message.TransactionHashes.Length > DBFTPlugin.System.Settings.MaxTransactionsPerBlock) return;
```

Comment:
View change?

Reply:
Can be, @shargon. I think it makes sense, however, it would be more like a case that will never happen with honest nodes...aehauhuea
Maybe like this is enough for now.

Reply:
I think that we should change the view if the policy it's different

Reply:
I think we can keep this and just wait for timeout. In other cases where verification fails such as tx verification failure, Block version failure, etc. it also return without changing view

Reply:
Why wait for timeout if we have a different policy and I don't want to sign your message?